src/background/feeder/e2e-session.feeder.ts

```diff
@@ -72,20 +45,10 @@ export class E2ESessionFeeder {
             throw new Error('E2E Session Feeder: Extension ID is undefined.')
           }
 
-          if (action === Action.REGISTER) {
+          if (action === MessageKeys.REGISTER) {
             response.data = this.handleRegistration(sender)
-          } else if (action === Action.WEB2_HELPER_FAKE_BEE_API_ADDRESS) {
-            response.data = this.handleWeb2FakeBeeApiAddress(request)
-          } else if (action === Action.WEB2_HELPER_FAKE_BZZ_ADDRESS) {
-            response.data = this.handleWeb2FakeBzzAddress(request)
-          } else if (action === Action.WEB2_HELPER_BEE_ADDRESS) {
+          } else if (action === MessageKeys.BEE_API_URLS) {
             response.data = await this.handleWeb2BeeAddress()
-          } else if (action === Action.BZZ_LINK_PROTOCOL_TO_FAKE_URL) {
-            response.data = this.handleBzzLinkProtocolToFakeUrl(request)
-          } else if (action === Action.BZZ_LINK_LINK_URL_TO_FAKE_URL) {
-            response.data = this.handleBzzLinkUrlToFakeUrl(request)
-          } else if (action === Action.BZZ_LINK_URL_TO_FAKE_URL) {
-            response.data = this.handleBzzUrlToFakeUrl(request)
           } else {
             throw new Error(`E2E Session Feeder: Unknown action ${action}`)
           }
```

Comment:
If you mean this line: https://github.com/ethersphere/swarm-extension/blob/4b9d8a5caf3038ded1c9998a18eb77aded3114e3/src/background/feeder/e2e-session.feeder.ts#L56

This will convert the error into a string, whatever the error is.